R/match_geo_names.R

```diff
@@ -0,0 +1,94 @@
+#' @title Match geography codes with their names
+#'
+#' @description \code{match_area_names} takes a geography code and brings back the
+#' name of the area.
+#'
+#' @details It uses the standard 9 digit codes for matching with the names.
+#' It can match with any of the current and previous versions of the codes (e.g
+#' 2014 and 2019 Health Boards).
+#'
+#' It works for Health Boards, Council Areas, Health and Social Care Partnerships,
+#' Intermediate Zones, Data Zones (2001 and 2011), Electoral Wards,
+#' Scottish Parliamentary Constituencies, UK Parliamentary Constituencies,
+#' Travel to work areas, National Parks, Commmunity Health Partnerships,
+#' Localities (S19), Settlements (S20) and Scotland
+#'
+#' @param dataset Data frame/tibble that contains the geography code variable.
+#' @param code_var The variable that contains the geography codes. Needs to be
+#' between quotes.
+#'
+#' @examples
+#' test_df <- data.frame(code = c("3", "S01002363", "S01004303", "S02000656",
+#' "S02001042", "S08000020", "S12000013", "S12000048",
+#' "S13002522", "S13002873", "S14000020", "S16000124", "S17000012"))
+#'
+#' match_area_names(dataset = test_df, code_var = "code")
+#' test_df %>% match_geo_names("code")
+#'
+#' @export
+match_area_names <- function(dataset, code_var) {
```

Comment:
The filename needs to change to match this!

Reply:
done